bentoml/_internal/server/__init__.py

```diff
@@ -131,6 +193,33 @@ def serve_production(
 ) -> None:
     working_dir = os.path.realpath(os.path.expanduser(working_dir))
     svc = load(bento_identifier, working_dir=working_dir)
+    serve_info = get_serve_info()
+    current_pid = os.getpid()
+    bento_creation_timestamp = None
+
+    # Track first time
+    if svc.bento is not None:
+        bento = svc.bento
+        bento_creation_timestamp = bento.info.creation_time.isoformat()
+        event_properties = {
+            "production": True,
+            "bento_identifier": bento_identifier,
+            "bento_creation_timestamp": bento_creation_timestamp,
+            **serve_info,
+        }
+    else:
+        # In this case serving from a file/directory with --production
+        event_properties = {
+            "production": True,
+            "bento_tag": bento_identifier,
+            **serve_info,
+        }
+
+    track(
+        BENTO_SERVE_TRACK_EVENT_TYPE,
```

Comment:
to make it easier for analytics, can we separate serve_production and serve_development to two different event types?

Reply:
we actually have a `bento_serve_scheduled_track` and `bento_serve_init_track`